kcqrs-core/src/main/kotlin/com/clouway/kcqrs/core/Command.kt

```diff
@@ -6,4 +6,4 @@ package com.clouway.kcqrs.core
  * 
  * @author Miroslav Genov (miroslav.genov@clouway.com)
  */
-interface Command
\ No newline at end of file
+interface Command<T>
```

Comment:
Suggest: 
If is introduced new interface will have compatibility with previous versions and will be possible smooth migration.